packages/new_stats/src/components/network_filter.vue

```diff
@@ -85,12 +84,12 @@ const networks = ref([
 ]);
 
 // initial state; all network selected
-const selectedNetworks = ref([Network.Dev, Network.Main, Network.Test]);
+const selectedNetworks = ref(["dev", "main", "test"]);
```

Comment:
why u didn't add `qa` ?

Reply:
I'm not entirely sure why qa wasn't added from the start, @0oM4R do you have any idea why?

Reply:
Yes we don't have to include qa net on our stats, the required network is only dev, test and main

Reply:
Thank youu, i didn't know that.